pkg/controller/drain.go

```diff
@@ -303,7 +313,7 @@ func (o *DrainOptions) getPodsForDeletion() (pods []api.Pod, err error) {
 		return []api.Pod{}, errors.New(fs.Message())
 	}
 	if len(ws) > 0 {
-		fmt.Fprintf(o.ErrOut, "WARNING: %s\n", ws.Message())
+		glog.Warningf("%s", ws.Message())
```

Comment:
Can you please check if o.ErrOut is really required to be removed?

Reply:
Reverted the changes W.R.T to fmt.Fprintf. As this is used by the logger to log messages.